### PR TITLE
Add cases of Cloud Run kind into DetermineAppHealthStatus

### PR DESCRIPTION
**Pull request by knanao**
**What this PR does / why we need it**:

**Which issue(s) this PR fixes**:

Fixes #

**Does this PR introduce a user-facing change?**:

```release-note
NONE
```

### CODE REVIEW

File: pkg/model/application_live_state.go [hunk 1]
```diff
@@ -60,23 +60,46 @@ func (s KubernetesResourceState) HasDiff(a KubernetesResourceState) bool {
 }
 
 // DetermineAppHealthStatus updates its own health status, which is determined based on its resources status.
+// TODO: Determine health state of other than k8s and cloud run app
 func (s *ApplicationLiveStateSnapshot) DetermineAppHealthStatus() {
 	switch s.Kind {
+	case ApplicationKind_TERRAFORM,
+		ApplicationKind_LAMBDA,
+		ApplicationKind_ECS:
```
Comment (nghialv) [inline pkg/model/application_live_state.go:68]: nit: Remove this for now.

File: pkg/model/application_live_state.go [hunk 2]
```diff
@@ -60,23 +60,46 @@ func (s KubernetesResourceState) HasDiff(a KubernetesResourceState) bool {
...
-			}
+		s.determineKubernetesAppHealthStatus()
+	case ApplicationKind_CLOUDRUN:
+		s.determineCloudRunAppHealthStatus()
+	}
+	return
+}
+
+func (s *ApplicationLiveStateSnapshot) determineKubernetesAppHealthStatus() {
+	k := s.Kubernetes
```
Comment (nghialv) [inline pkg/model/application_live_state.go:78]: nit: IMO, "app" is more understandable than "k" 🤚

File: pkg/model/application_live_state.go [hunk 3]
```diff
@@ -60,23 +60,46 @@ func (s KubernetesResourceState) HasDiff(a KubernetesResourceState) bool {
...
 		}
-		s.HealthStatus = status
-	default:
-		// TODO: Determine health state of other than k8s app
+	}
+	s.HealthStatus = status
+}
+
+func (s *ApplicationLiveStateSnapshot) determineCloudRunAppHealthStatus() {
+	c := s.Cloudrun
```
Comment (nghialv) [inline pkg/model/application_live_state.go:93]: nit: IMO, "app" is more understandable than "c" 🤚

Comment (knanao) [reply pkg/model/application_live_state.go]: TYVM:D
PTAL https://github.com/pipe-cd/pipecd/pull/3280/commits/ee9ff7d8edfc9ad366d45ce29c83d950a03625a5 when you can check it.

**Comment by nghialv** [comment]
Nice. Thank you.
/lgtm

**Pull request merged**